src/chrome/chromeConnection.ts

```diff
@@ -27,6 +27,7 @@ export interface ITarget {
     type: string;
     url?: string;
     webSocketDebuggerUrl: string;
+    version: Promise<ProtocolSchema>;
```

Comment:
Do you think we should wait on getting the version so it can just a ProtocolSchema instead of a promise?